### ISSUE TITLE
Feature Request: Handle forwarded messages in separate threads

### ISSUE THREAD

**Issue by samtuke** [Foundry376/Mailspring#1955]
When a message is forwarded, it appears in the same thread, which will then reply to the person you forwarded the message to, instead of to the original sender, when you press reply.

Use case:

- Important mail arrives
- You forward to colleagues for their opinion
- You go back to the original message, hit reply, intending to reply to the sender of the initial mail
- Instead, you send a message to whoever to you last forwarded it to

In my case I didn't realise this had happened for a week, and some important document didn't arrive.

More generally, mixing threads with forwards makes their history hard to follow, especially when you receive replies to those forwards.

UX would be improved if the original message and replies thereto, were kept separate from forwarded copies and replies thereto. Ideally each forward should be a separate thread -- you may want separate confidential discussions about a forwarded message with different people, and not want to risk mixing them or sending confidential info to the wrong person.

**Comment by CodeMouse92** [comment]
We are in the process of migrating issues to Discourse, which can better facilitate discussion and discovery, and so GitHub Issues can focus on issues that are confirmed and slated for resolution in the near term. [Learn more about the changes here.](https://community.getmailspring.com/t/mailspring-community-transition/38)

As part of this, we've migrated this issue to Discourse:

https://community.getmailspring.com/t/handle-forwarded-messages-in-separate-threads/890

Please consider joining that community and continuing the discussion there! **Votes on the feature suggestion on Discourse will increase the likelihood we implement this.**

@samtuke: if you join and reply to the issue, the moderators can make an effort to reassign your posts to you, so you get the credit for them.

We're closing and locking the issue here as part of this migration. Rest assured, this doesn't mean the issue is being discarded or ignored.

We hope to see you on Discourse soon!

-The Mailspring Team
